db/schema_0.sql

```diff
@@ -210,6 +210,7 @@ CREATE TABLE `plan_notification` (
   `step` int(11) NOT NULL,
   `template` varchar(255) DEFAULT NULL,
   `target_id` bigint(20),
+  `optional` TINYINT(1) NOT NULL DEFAULT '0',
```

Comment:
I'd prefer this to be BOOLEAN, so we can do True/False in our REST API calls instead of 0/1.

Reply:
I defined it as a tinyint for consistency, BOOLEAN should be completely interchangeable https://dev.mysql.com/doc/refman/8.0/en/other-vendor-data-types.html